addons/markers/functions/fnc_initInsertMarker.sqf

```diff
@@ -29,26 +30,28 @@
     };
 
     //BIS Controls:
-    private _text = _display displayctrl 101;
-    private _picture = _display displayctrl 102;
-    private _channel = _display displayctrl 103;
-    private _buttonOK = _display displayctrl 1;
-    private _buttonCancel = _display displayctrl 2;
-    private _description = _display displayctrl 1100;
-    private _title = _display displayctrl 1001;
-    private _descriptionChannel = _display displayctrl 1101;
+    private _text =                 _display displayctrl IDC_INSERT_MARKER;
```

Comment:
why do you add those spaces?

Reply:
I find it more readable, but I can revert it otherwise.

Reply:
I think it breaks the readability but that a maintainer should decide. I think we have it nowhere else in ace. I am also not sure if it is against coding style.

Reply:
We usually don't do that here.

Reply:
Though is it something that I will have to change for this to be approved?

Reply:
Personally I think yes, do we have any guidelines on stuff like this? @jonpas ?

Reply:
informal slack poll had mixed to negative results for this type of formatting
probably ok to leave in the PR, but it might get cleaned up in later refactoring

Reply:
Resolving the conversation then.